### DISCUSSION TITLE
Asp.Net MVC how to schedule with Quartz?

### DISCUSSION THREAD

**Discussion by raysefo** [quartznet/quartznet#1811] [Q&A]
Hi There,
I was looking for a task scheduler for my web application. A friend suggested using Quartz. I wonder if I can set parameters to the task and start/stop it from this web application? Is there any tutorial that I can follow?

I really appreciate any help you can provide.
Thank you.

**Comment by lahma** [comment]
There's an example for ASP.NET Core at least: https://github.com/quartznet/quartznet/tree/main/src/Quartz.Examples.AspNetCore . If you're running on full framework this might not apply that well.
